appraisal.lic

```diff
@@ -13,21 +13,39 @@ class Appraisal
     settings = get_settings
```

Comment:
💡 Minor: I _think_ these three declarations could be moved below the arg parsing.